### PR TITLE
tier-2_rados_basic_regression: assimilate configurations from ceph.conf

### PR DESCRIPTION
**Pull request by pdhiran**
Test workflow to move the daemon configurations set in ceph.conf file into ceph monitor configuration database. Document : https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/5/html/configuration_guide/the-basics-of-ceph-configuration#the-ceph-configuration-database_conf

Signed-off-by: Pawan Dhiran <pdhiran@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [x] Create a test case in Polarion reviewed and approved.
- [x] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [x] Review the automation design
- [x] Implement the test script and perform test runs
- [x] Submit PR for code review and approve
- [x] Update Polarion Test with Automation script details and update automation fields
- [x] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by pdhiran** [comment]
Pass logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-PGAI56

**Comment by pdhiran** [comment]
Pass Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-X8TSPA

File: tests/rados/test_config_assimilation.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    log.info("Assimilating the contents of ceph.conf into mon config")
+    cmd = "ceph config assimilate-conf -i /etc/ceph/ceph.conf -o /etc/ceph/assimilate-op.txt"
+    client_node.exec_command(cmd=cmd, sudo=True)
+
+    # Verifying the configs set via the CLI in the mon config database.
+    test_config = config.get("Verify_config_parameters")
+    for conf in test_config["configurations"]:
+        for entry in conf.values():
+            if not mon_obj.verify_set_config(**entry):
+                log.error(f"Error setting config {entry}")
```
Comment (sunilkumarn417) [inline tests/rados/test_config_assimilation.py:65]: raise exception, lets not return 1 for error.

File: tests/rados/test_config_assimilation.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    )
+
+    # removing the additional configurations set in the mon database
+    for conf in test_config["configurations"]:
+        for entry in conf.values():
+            if not mon_obj.remove_config(**entry):
+                log.error(
+                    f"Failed to remove config : {conf} from the mon config database"
+                )
+                return 1
```
Comment (sunilkumarn417) [inline tests/rados/test_config_assimilation.py:80]: same as above

**Review by sunilkumarn417 — APPROVED** [review]

File: tests/rados/test_config_assimilation.py (hunk 1 above)
Comment (pdhiran) [reply tests/rados/test_config_assimilation.py:65]: Will make with the Next PR.

**Comment by pdhiran** [comment]
Will make changes WRT the return code in the next PR.

**Pull request merged**